gnd/src/main/java/com/google/android/gnd/model/feature/Feature.java

```diff
@@ -39,8 +52,12 @@
   @Nullable
   public abstract String getCaption();
 
+  @Nullable
```

Comment:
This may be a good use of polymorphism, since PointFeature and GeoJsonFeature are mutually exclusive feature types. Wdyt?

Note that I don't know whether we will want to support drawing of multi-polygons (vs one polygon per feature) in the future; imported GeoJson might be treated differently from polygons that can be edited in the UI (at least for now), which is why I suggested calling the subclass specifically `GeoJsonFeature` instead of something like `MultiPolygonFeature`.

Reply:
This is what I went for in the first place. But then I realized that 'point' is a non-null property right now.
So this change cannot be made until point becomes optional wherever it is being used.

Reply:
That's why I was suggesting subclassing Feature; PointFeature could have `@NonNull getLocation()`, and GeoJsonFeature could have `@NonNull getPolygons()`, for example. Would that work? Might require some changes here and there though, but it would allow us to enforce the presence of a point or polygon at compile time instead of runtime.